src/pg-storage-client.ts

```diff
@@ -497,4 +499,20 @@ export default class PGStorageClient {
         await this.writeMiscProperty(name, defaultValue);
         return defaultValue;
     }
+
+    async fetchAllGuildSettings(): Promise<Record<string, Partial<Record<number, number>>>> {
```

Comment:
Should use `GuildSetting`, not `number`. Or just use reference type (see other comment)